src/Language/PureScript/Ide/SourceFile.hs

```diff
@@ -68,31 +66,39 @@ getImportsForFile fp = do
         unwrapImportType (P.Hiding decls)   = P.Hiding (map unwrapPositionedRef decls)
         unwrapImportType P.Implicit         = P.Implicit
 
-getDeclPosition :: P.Module -> Text -> Maybe P.SourceSpan
-getDeclPosition m ident = getFirst (foldMap (match ident) decls)
+-- | Given a surrounding Sourcespan and a Declaration from the PS AST, extracts
+-- definition sites inside that Declaration.
+extractSpans
+  :: P.SourceSpan -- ^ The surrounding span
```

Comment:
You might like to move the comments to the next line. The `^` is meant to point upwards to the code above, I believe.